src/PKSim.Infrastructure/InfrastructureRegister.cs

```diff
@@ -251,6 +256,19 @@ private static void registerReportBuilders(IContainer container)
          container.Register<IReportBuilderRepository, ReportBuilderRepository>(LifeStyle.Singleton);
       }
 
+
+      private static void registerMarkdownBuilders(IContainer container)
```

Comment:
similar to our tex builder, we need to implement markdown builder. It should be much simple and I believe could potentially replace Tex in the future